src/GeometryInstancing/Program.cs

```diff
@@ -0,0 +1,261 @@
+﻿using System.IO;
+using System.Numerics;
+using Veldrid;
+using Veldrid.Sdl2;
+using Veldrid.StartupUtilities;
+using Veldrid.NeoDemo;
+
+namespace GeometryInstancing

```

Comment:
I think we should rename this to `InstancedQuads` or something, because `Geometry` might lead you to believe it uses Geometry shaders.